### PR TITLE
Node Information service was added

### PR DESCRIPTION
**Pull request by ahmedrns**

### CODE REVIEW

File: docs/config.example.yaml [hunk 1]
```diff
@@ -68,3 +67,10 @@ votera:
 
     # Endpoint of Agora
     votera_endpoint: http://dev-votera.bosagora.com:1337/
+
+################################################################################
+##                               Node server config                           ##
+################################################################################
+node:
+    # Endpoint of Agora
+    node_endpoint: https://v2.bosagora.io/
```
Comment (TrustHenry) [inline docs/config.example.yaml]: agora_endpoint: http://127.1.1.1:4567
You can refer to this without adding anything new.

File: src/modules/storage/LedgerStorage.ts [hunk 2]
```diff
@@ -516,6 +517,18 @@ export class LedgerStorage extends Storages {
             PRIMARY KEY(block_height, utxo_key(64))
         ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+        CREATE TABLE IF NOT EXISTS nodes
```
Comment (TrustHenry) [inline src/modules/storage/LedgerStorage.ts:520]: Please document it.

File: src/modules/nodeClient/NodeClient.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+     * The instance of the axios
+     */
+    private client: AxiosInstance;
+
+    constructor(endpoint: URL) {
+        this.endpoint = endpoint;
+        this.client = axios.create({
+            baseURL: endpoint.toString(),
+            // Timeout is in ms
+            timeout: 10000,
```
Comment (TrustHenry) [inline src/modules/nodeClient/NodeClient.ts:23]: Isn't this timeout causing delays in all API request responses for STOA?

Comment (ahmedrns) [reply src/modules/nodeClient/NodeClient.ts:23]: As like other services this will not effect the stoa APIs

File: docs/config.example.yaml [hunk 4]
```diff
@@ -68,3 +67,10 @@ votera:
 
     # Endpoint of Agora
     votera_endpoint: http://dev-votera.bosagora.com:1337/
+
+################################################################################
+##                               Node server config                           ##
+################################################################################
+node:
+    # Endpoint of Agora
+    node_endpoint: http://127.1.1.1:4567
```
Comment (TrustHenry) [inline docs/config.example.yaml]: Please remove this setting.
server.agora_endpoint <- You can refer to this instead.

File: docs/Database_Schema.md [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
-| hash              | TINYBLOB  |    | Y        |          | The hash of the current block |
-| prev_block        | TINYBLOB  |    | Y        |          | The hash of the previous block |
-| validators        | TEXT      |    | Y        |          | Bitfield containing the validators' key indices which signed the block |
-| merkle_root       | TINYBLOB  |    | Y        |          | The hash of the merkle root of the transactions|
-| signature         | TINYBLOB  |    | Y        |          | Schnorr multisig of all validators which signed this block |
-| tx_count          | INTEGER   |    | Y        |          | The number of transactions in the block|
-| enrollment_count  | INTEGER   |    | Y        |          | The number of enrollments in the block|
-| time_offset       | INTEGER   |    | Y        |          | Block seconds offset from Genesis Timestamp |
-| time_stamp        | INTEGER   |    | Y        |          | Block unix timestamp |
+| Column           | Data Type | PK  | Not NULL | Default | Description                                                            |
```
Comment (TrustHenry) [inline docs/Database_Schema.md]: Please do not modify the space spacing of the unmodified document.

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**